grakn-engine/src/main/java/ai/grakn/engine/util/EngineID.java

```diff
@@ -65,27 +67,8 @@ public static EngineID me() {
         return EngineID.of(value);
     }
 
-    public String value() {
-        return value;
-    }
-
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
-
-        EngineID engineID = (EngineID) o;
-
-        return value.equals(engineID.value);
-    }
-
-    @Override
-    public int hashCode() {
-        return value.hashCode();
-    }
-
     @Override
     public String toString() {
-        return escapeJava(value);
+        return escapeJava(getValue());
     }
```

Comment:
I would just delete this `toString` completely. `AutoValue` makes a pretty good one.

Reply:
I will try kill it then. I thought we would need the `escapeJava ` part